mailpoet/lib/Automation/Engine/Builder/CreateWorkflowFromTemplateController.php

```diff
@@ -2,6 +2,7 @@
 
 namespace MailPoet\Automation\Engine\Builder;
 
+use _PHPStan_ae8980142\Nette\InvalidStateException;
```

Comment:
Ups 😄 Your IDE did this, I guess.

Reply:
:facepalm: Addressed in [17e2f97](https://github.com/mailpoet/mailpoet/pull/4362/commits/17e2f979db39561b3ba63b702b08ce7b4c476540)